frontend/src/views/SqlEditor/TablePanel/TableView.vue

```diff
@@ -153,16 +153,18 @@ const handleExportBtnClick = (format: "csv" | "json") => {
 
   if (format === "csv") {
     let CSVContent = "";
-    CSVContent += columns.value.map((item) => String(item.key)).join(",");
+    CSVContent += columns.value
+      .map((item) => JSON.stringify(item.key))
+      .join(",");
     CSVContent += "\n";
 
     for (const d of data.value) {
       const temp: any[] = [];
       for (const k in d) {
         temp.push(d[k]);
       }
-      CSVContent += temp.map((item) => String(item)).join(",");
-      CSVContent += "\n";
+      CSVContent += temp.map((item) => JSON.stringify(item)).join(",");
```

Comment:
I'm not sure if this can entirely fix the separator issue, we'd better extract this as a pure function and add several tests for it later.
Currently I'm okay with it cause we can improve this later.

Reply:
This still can't handle all cases, but does better than `String(value)`. I will improve it in the next version.